sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Renamed `KeyPropertiesPageResult` to `KeyPropertiesPagedResponse`.
 - Renamed `DeletedKeyPageResult` to `DeletedKeyPagedResponse`.
 - Changed the container for `KeyOperations` from `std::list` to `std::vector` within `CreateKeyOptions` and `UpdateKeyProperties()`.
+- Changed the return type of `CrytographyClient` APIs like `Encrypt()` to return `Response<T>` rather than the `T` directly.
```

Comment:
Addresses the change from https://github.com/Azure/azure-sdk-for-cpp/pull/2638#pullrequestreview-707897241